PicoProducer/python/analysis/ModuleMuMu.py

```diff
@@ -125,6 +126,9 @@ def analyze(self, event):
     muon1.tlv    = muon1.p4()
     muon2.tlv    = muon2.p4()
     self.out.cutflow.fill('pair')
+
+    if self.jetveto(event): return False 
```

Comment:
I believe this would break in run2. Maybe one can modify the jetvetomap tool in such a way that it returns always True when era is in run2? @IzaakWN

Reply:
I think this is the only comment not being addressed, can you do here something like:
`if "202" in self.year:
     if self.jetveto(event): return False 
`